src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildWrapper.java

```diff
@@ -308,6 +310,18 @@ public Environment setUp(final AbstractBuild build, Launcher launcher, final Bui
                 listener.getLogger().println("Error checking Sauce Connect run condition");
                 throw new IOException(e);
             }
+
+            EnvVars env = new EnvVars();
+            try {
+                env = build.getEnvironment(listener);
+            } catch (IOException e) {
+                listener.getLogger().println("Error getting environment variables");
```

Comment:
Here I was following the convention elsewhere laid out in SauceOnDemandBuildWrapper, so as to prevent enforcing my own preferred style of error / message management.